autogluon/inference/docker/0.7/py3/Dockerfile.cpu

```diff
@@ -54,14 +31,27 @@ RUN pip install --no-cache-dir -U --trusted-host pypi.org --trusted-host files.p
  && pip install --no-cache-dir -U numpy numba \
  && pip install --no-cache-dir -U autogluon==${AUTOGLUON_VERSION} \
  && mim install -q mmcv-full \
- && pip install --no-cache-dir -U mmdet 
+ && pip install --no-cache-dir -U mmdet
 
 # Removing GluonTS nursery/tsbench package - it is not used in training/inference and have security vulnerabilities
 RUN rm -rf /usr/local/lib/python3.9/dist-packages/gluonts/nursery/tsbench
 
 # Remove these to address security issues; these are not used if only python package is used
+# flask Flask-RESTful nvgpu are reqiurements for torch-serve
 RUN rm -rf /usr/local/lib/python3.9/dist-packages/ray/jars \
- && rm -rf /opt/conda/lib/python3.9/site-packages/ray/jars
+ && rm -rf /opt/conda/lib/python3.9/site-packages/ray/jars \
+ && pip install -U flask Flask-RESTful nvgpu \
+ && conda update -y requests \
+ && pip install -U botocore pyopenssl google-auth awscli \
```

Comment:
are these needed for any specific reason?

Reply:
those were flagged as security vulnerabilities (dependencies are coming from torch-serve)